### DISCUSSION TITLE
How to connect to Amazon Memory DB in tls insecure mode

### DISCUSSION THREAD

**Discussion by anand0761** [redisson/redisson#5058] [Q&A] [answered]
I am able to connect to memory db cluster using below command, trying to replicate the same via redission client but not able to connect please suggest the way forward. 
`redis-cli -c --user anand --askpass -h clustercfg.testcluster.amazonaws.com --tls --insecure`

```
Config config = new Config();
    config
    .useClusterServers()
    .addNodeAddress("redis://clustercfg.testcluster.amazonaws.com:6379")
    .setSslProtocols(new String[]{"TLSv1.3", "TLSv1.2", "TLSv1.1, TLSv1"})
    .setSslProvider(SslProvider.valueOf("JDK"))
    .setSslEnableEndpointIdentification(true)
    .setUsername("ANAND")
    .setPassword("pwd");
```

**Comment by mrniko** [comment, marked as answer]
you need to use `rediss://` prefix in url instead of `redis://`
